povertydata.rb

```diff
@@ -0,0 +1,90 @@
+class County
+  attr_accessor :county, :state, :percent, :child_poverty_count, :median_income
+
+  def initialize(county, state, percent, child_poverty_count, median_income)
+    @county = county
+    @state = state
+    @percent = percent
+    @child_poverty_count = child_poverty_count
+    @median_income = median_income
+  end
+
+  def self.all
+    puts "Please enter WA or CA for information:  "
+    state = gets.chomp.upcase
+    if state == "WA"
+      @file= File.open('est14_WA.txt')
+    elsif state == "CA"
+      @file= File.open('est14_CA.txt')
+    end
+    array = []
+    @file.each do |row|
+      array.push(
+      County.new(row[193..237], row[239..241], row[76..79], row[49..56], row[133..138]))
+    end
+    return array
+  end
+
+  def self.highest_poverty_WA
```

Comment:
This is interesting - I don't think there is anything to prevent the user from calling this method and then selecting for CA, right?